test/library/packages/Python/correctness/virtualEnv/sub_test

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+python3 -m venv .venv
```

Comment:
this should use `find-python.sh`, similar to the logic in `checkAndInstallPackage.sh`